match2/commons/match_group_display_helper.lua

```diff
@@ -1,9 +1,13 @@
+--
+-- @author QuadratClown for Liquipedia
+--
+
 local Array = require('Module:Array')
 local FnUtil = require('Module:FnUtil')
-local Json = require('Module:Json')
+local Json = require("Module:Json")
 local LuaUtils = require('Module:LuaUtils')
 local Table = require('Module:Table')
-local utils = require('Module:LuaUtils')
+local utils = require("Module:LuaUtils")
```

Comment:
Any reason why we import Module:LuaUtils twice?

Reply:
The code I added use `LuaUtils`, existing code used `utils` and I didn't change it. Since `LuaUtils` is deprecated both will be removed pretty soon.